example.yml

```diff
@@ -109,7 +118,7 @@ awards:
     url:
     location: City of London, United Kingdom
     highlights:
-      # - Champion in the Division A District X Speech Evaluation Contest of Toastmasters International in London.
+      - Champion in the Division A District X Speech Evaluation Contest of Toastmasters International in London.
```

Comment:
Could you also please delete this line? I forgot I had this still commented out to test awards without a description.

Reply:
Alright I will do that.